UnityProject/Assets/Scripts/UniMic/Runtime/Mic.cs

```diff
@@ -3,6 +3,9 @@
 using System.Collections;
```

Comment:
if the code for UniMic is in the codebase, what stops forks from modifying this very code and bypass the secure stuff code?

Reply:
well nothing can access the microphone, it will go through secure stuff to access the microphone